python_src/src/lamp_py/performance_manager/l1_cte_statements.py

```diff
@@ -237,11 +237,7 @@ def trips_for_metrics_subquery(
             sa.func.lead(rt_trips_sub.c.vp_move_timestamp)
             .over(
                 partition_by=rt_trips_sub.c.vehicle_id,
-                order_by=sa.func.coalesce(
-                    rt_trips_sub.c.vp_move_timestamp,
-                    rt_trips_sub.c.vp_stop_timestamp,
-                    rt_trips_sub.c.tu_stop_timestamp,
-                ),
+                order_by=rt_trips_sub.c.vp_move_timestamp,
```

Comment:
was this change necessary as well if we're running the VACUUM?

Reply:
This change wasn't necessary, but looking at the field that's being calculated, and how it's used. I don't believe the coalesce is needed. 

I think this business logic was a little bit of a hold over from before Ops Analytics decided they wanted all headways as departure to departure calculations.